forc/src/utils/defaults.rs

```diff
@@ -28,6 +29,7 @@ edition = "2021"
 license = "Apache-2.0"
 name = "{project_name}"
 version = "0.1.0"
+rust-version = "1.58.0"
```

Comment:
We should not hard-code the version here, but instead use the `forc_rustc_version` function to retrieve the compatible version.

Reply:
My comment here still stands. It looks like you've removed the `forc_rustc_version` function, but that's what we should be using to determine what version to use in this field.